src/amo/components/MastHead.scss

```diff
@@ -0,0 +1,66 @@
+@import "~amo/css/inc/vars";
+
+#tabzilla {
+  left: 14px;
```

Comment:
Good call. I will check that out - I added this because of the mock, but it seems backwards to the typical way.